models/prices/prices_usd_latest_historical.sql

```diff
@@ -20,5 +20,6 @@ SELECT
 FROM {{ source('prices', 'usd') }} pu
 {% if is_incremental() %}
     WHERE minute >= date_trunc('day', now() - interval '2' day)
+    AND pu.contract_address != 0xd31fcd1f7ba190dbc75354046f6024a9b86014d7 -- remove bad price feed
```

Comment:
@safeintern i've already removed this token from the upstream source that loaded it to `prices.usd`, seen here:
https://github.com/duneanalytics/spellbook/pull/5826

where are you seeing the downstream effects of this pricing issue? is it this spell itself or another spell downstream?

please help by posting query evidence of the issue, so i can try to help fix. we likely don't need another fix/merge, but rather a refresh of a spell (or spell lineage)